mantidproject/mantid
# New load widget for Elwin Tab

**Description of work.**
A new loader for Elwin has been developed. Users can choose whether to upload from a file or use an exisiting workspace. 

**To test:**
1. Open Mantid and navitgate to Interfaces/Indirect/Data Analysis
2. The Elwin tab is the default when opening the Data Analysis GUI. Please check the GUI looks as per image below

![Screenshot (16)](https://user-images.githubusercontent.com/55837273/129684414-da1f2354-5927-475d-bcba-fcf3bdc7f250.png)

3. Download the Elwin dataset from PR #26631 
4. Click 'Browse' and navigate to where you downloaded irs26176_graphite002_red.nxs. Choose it to load file. Data should load in the plotting area below and no error messages should occur.
5. Click Run towards the bottom of the screen. 3 new workspaces should have been added to the Workspace list ending with '_elf', '_eq', and '_eq2' respectively. No error messages should occur.
6. Close the GUI
7. Reopen the GUI
8. Change the drop down on the left from File to Workspace. The GUI interface should change and look as per the image below.

![Screenshot (17)](https://user-images.githubusercontent.com/55837273/129684440-4e4860b7-c423-4202-bc46-98303d0925ef.png)

9. Click on Add Workspace
10. Change the dropdown to Workspace and you should be able to add irs26176_graphite002_red. Tick All Spectra and press Add.
11. Click Close on the Workspace Dialog. The Table should now have all the Spectra listed.
12. Change the Spectrum in the plotting area below using the up/down button toggles. 
13.  Click Run towards the bottom of the screen. 3 new workspaces should have been added to the Workspace list ending with '_elf', '_eq', and '_eq2' respectively. No error messages should occur.


Fixes #29536 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
